src/modules/dashboard/components/ExpenditurePage/Split/SplitUnequal/SplitUnequal.tsx

```diff
@@ -0,0 +1,92 @@
+import { useFormikContext } from 'formik';
+import React from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+// import classNames from 'classnames';
+
+import { FormSection, Input, TokenSymbolSelector } from '~core/Fields';
+import { Colony } from '~data/index';
+
+import styles from './SplitUnequal.css';
+
+import { ValuesType } from '~pages/ExpenditurePage/ExpenditurePage';
+
+const MSG = defineMessages({
+  reserve: {
+    id: 'dashboard.ExpenditurePage.Split.SplitUnequal.reserve',
+    defaultMessage: 'Reserve',
+  },
+  amountLabel: {
+    id: 'dashboard.ExpenditurePage.Split.amountLabel',
+    defaultMessage: 'Amount',
+  },
+});
+
+const displayName = 'dashboard.ExpenditurePage.Split';
```

Comment:
This has the same displayName as Split.tsx. How about using 'dashboard.ExpenditurePage.SplitUnequal'?